Delicate-Butterflies/Bangazon-Cli
# Regex validation on user input

Related Issue: #52 

Description: Validation of user inputs on Create new user and add customer product

Any other PR Testing Steps:
1. select option 4
- should not be able to select a product-type out of the range displayed
- unit price can be a float and accepts only numbers
- only numbers accepted in 'how many for sale'

2. select option 1
- first name should accept only letters and . (just in case they have jr. in their name)
- last name should accept only letters and . (just in case they have jr. in their name)
- city name only accepts letters
- state code only accepts 2 letter code
- postal code only accepts 5 digits
